src/snowflake/snowpark/column.py

```diff
@@ -761,8 +1361,8 @@ def _to_expr(cls, expr: Union[ColumnOrLiteral, Expression]) -> Expression:
             return Literal(expr)
```

Comment:
are we missing _emit ast for literal

Reply:
We do not emit asts for expressions (as we consider them an internal API, which will be moved to the TCM). Given Literal inherits from Expression, there is not _emit_at parameter for it.

I am curious, why would you want to add it here for literal here?